__tests__/app.test.js

```diff
@@ -143,17 +143,18 @@ describe("NCNews API testing", () => {
               votes: 100,
               article_img_url:
                 "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+              comment_count: "11",
```

Comment:
ideally for ticket 11, we shouldn't have to amend previous tests to be able to add in the comment_count - this means your earlier tests were brittle. Using something like object.containing would prevent us from having to change tests if later on the down the line we add a new column. Something to consider going forward, especially if you were working on a massive project 😊